esm-samples/jsapi-custom-workers/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: {
+    index: ['./src/index.css', './src/index.js']
+  },
+  node: false,
+  output: {
+    path: path.join(__dirname, 'dist'),
+    chunkFilename: 'chunks/[id].js',
+    publicPath: '',
+    clean: true
+  },
+  devServer: {
+    contentBase: path.join(__dirname, 'dist'),
+    compress: true,
```

Comment:
More curious about this one - I haven't tested it, does compression on dev slow down the builds?

Reply:
this is for the dev server during `npm run dev`, it doesn't impact build times.